src/v/systems/contract/Contract.java

```diff
@@ -0,0 +1,112 @@
+package v.systems.contract;
+
+import org.bitcoinj.core.Base58;
+import v.systems.error.SerializationError;
+import v.systems.serialization.BytesSerializable;
+import v.systems.utils.BytesHelper;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Contract implements BytesSerializable {
+    private String languageCode;
+    private Integer languageVersion;
+    private List<String> triggers;
+    private List<String> descriptors;
+    private List<String> stateVariables;
+    private ContractTextual textual;
+
+    public Contract() {
+        languageCode = "vdds";
+        triggers = new ArrayList<String>();
+        descriptors = new ArrayList<String>();
+        stateVariables = new ArrayList<String>();
+        textual = new ContractTextual();
+    }
+
+    @Override
+    public byte[] toBytes() throws SerializationError {
+        try {
+            byte[] triggerBytes = serializeField(triggers);
+            byte[] descriptorBytes = serializeField(descriptors);
+            byte[] stateVarBytes = serializeField(stateVariables);
+            byte[][] result = new byte[][]{
```

Comment:
Why not use `Bytes` in Guava, or `ByteBuffer`?

Reply:
`Guava` is great library. However, we only use convert integer to bytes and concat functions from it. `BytesHelper.java`(which only 80 Lines) can meet the requirement. I'll consider using `Guava` if we need more Guava function later. For this version, I just want to keep SDK in lightweight.